packages/react-inbox-next/README.md

```diff
@@ -0,0 +1,2 @@
+<!-- START doctoc -->
```

Comment:
is this intentional?

Reply:
yeah we have a readme script that appends all the markdown in the docs folder to create the root level readme with a TOC

Reply:
yeah I thought we had that but didn't realize its controlled thru code